its/src/test/java/com/sonar/it/vbnet/VbMainCodeCsTestCodeTest.java

```diff
@@ -65,14 +66,26 @@ public void hasIssues() {
   @Test
   public void logsContainInfo() {
     assertThat(buildResult.getLogsLines(l -> l.contains("WARN")))
-      .contains("WARN: This C# sensor will be skipped, because the current solution contains only TEST files and no MAIN files. " +
-        "Your SonarQube/SonarCloud project will not have results for C# files. " +
+      .contains("WARN: SonarScanner for .NET detected only TEST files and no MAIN files for C# in the current solution. " +
+        "Only TEST-code related results will be imported to your SonarQube/SonarCloud project. " +
+        "Many of our rules (e.g. vulnerabilities) are raised only on MAIN-code. " +
         "Read more about how the SonarScanner for .NET detects test projects: https://github.com/SonarSource/sonar-scanner-msbuild/wiki/Analysis-of-product-projects-vs.-test-projects");
+
     assertThat(buildResult.getLogsLines(l -> l.contains("INFO"))).contains(
       "INFO: Found 1 MSBuild VB.NET project: 1 MAIN project.",
       "INFO: Found 1 MSBuild C# project: 1 TEST project."
     );
     TestUtils.verifyNoGuiWarnings(ORCHESTRATOR, buildResult);
   }
 
+  @Test
+  public void metrics_are_imported_only_for_maint_proj()throws Exception {
+    assertThat(getMeasureAsInt("VbMainCsTest:VbMain", "files")).isNotNull();
```

Comment:
it would be more readable to pass the actual values for Test code, rather than "isNotNull" (initially I was wondering what the difference is)